project/CelebornBuild.scala

```diff
@@ -55,20 +43,81 @@ object CelebornCommonSettings {
   val guavaVersion = "14.0.1"
   val hadoopVersion = "3.2.4"
   val javaxServletVersion = "3.1.0"
+  val junitInterfaceVersion = "0.13.3"
+  // don't forget update `junitInterfaceVersion` when we upgrade junit
```

Comment:
seems no strong contract?

Reply:
1. The JUnit interface delineates its JUnit version compatibility matrix within its [documentation](https://github.com/sbt/junit-interface/#setup). Therefore, I think it's best to keep it consistent with the documentation.

| junit-interface version | JUnit version |
|------------------------|---------------|
| 0.13.2                 | 4.13.2        |
| 0.13.1                 | 4.13.1        |
| 0.13                   | 4.13          |
| 0.12                   | 4.12          |

2. As per the [SBT documentation](https://www.scala-sbt.org/1.x/docs/Testing.html#JUnit), if we're planning to upgrade from JUnit 4 to JUnit 5, we'll need to switch from using the sbt JUnit interface to the sbt-jupiter-interface.

Reply:
I see, it makes sense, thanks for explaination